packages/mui-base/src/useTab/useTab.types.ts

```diff
@@ -3,14 +3,18 @@ import { UseButtonRootSlotProps } from '../useButton';
 
 export interface UseTabParameters {
   /**
-   * You can provide your own value. Otherwise, we fall back to the child position index.
+   * You can provide your own value. Otherwise, it fall back to the child position index.
    */
   value?: number | string;
   /**
    * Callback invoked when new value is being set.
    */
   onChange?: (event: React.SyntheticEvent, value: number | string) => void;
   onClick?: React.MouseEventHandler;
+  /**
+   * If `true`, the component is disabled.
+   * @default false
```

Comment:
This is inherited from `useButton()`.

https://github.com/mui/material-ui/blob/b27c9e6249d35b9c01fb1ecc610a22f3c24ece1f/packages/mui-base/src/useButton/useButton.types.ts#L21-L26